app/src/main/java/com/nervousfish/nervousfish/data_objects/Profile.java

```diff
@@ -8,15 +8,22 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
+
+import nl.tudelft.ewi.ds.bankver.IBAN;
 
 /**
  * A Profile POJO to store a contact representing the user and the user's keypairs.
  */
-public class Profile implements Serializable {
+@SuppressWarnings({"PMD.UselessParentheses", "PMD.NullAssignment"})
+//1. The parentheses on line 104 are not useless.
+//2. We want iban to be immutable, but not a required field.
```

Comment:
Are you sure? The IBan seems to be a required field in the `CreateProfileActivity`...